live/src/lib/controller.utils.ts

```diff
@@ -0,0 +1,32 @@
+import { Router } from "express";
+import { registerControllers as registerRestControllers, registerWebSocketControllers } from "@plane/decorators";
+import "reflect-metadata";
+
+/**
+ * Register all controllers from the controllers array
+ * @param router Express router to register routes on
+ * @param controllers Array of controller classes to register
+ * @param dependencies Array of dependencies to pass to controllers
+ */
+export function registerControllers(router: Router, controllers: any[], dependencies: any[] = []): void {
```

Comment:
Can we use the existing function from decorators packages?

Reply:
Hey @Prashant-Surya, we're indeed using those only, but stitching the web socket and rest controllers exposed from there together there that's all